src/services/assetService.ts

```diff
@@ -77,10 +77,7 @@ export class AssetService {
             const ocrHeight = ocrExtent[3] - ocrExtent[1];
             const result = [];
             for (let i = 0; i < arr.length; i += 2) {
-                result.push([
-                    (arr[i] / ocrWidth),
-                    (arr[i + 1] / ocrHeight),
-                ]);
+                result.push((arr[i] / ocrWidth), (arr[i + 1] / ocrHeight));
```

Comment:
why do we need this change?

Reply:
this is fix the boundingBoxes data issue in auto-labeling